setup.py

```diff
@@ -0,0 +1,39 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="karoo_gp",
+    version="2.3",
+    author="Kai Staats",
+    author_email="github@overthesun.com",
+    description="Use Genetic Programming for Classification and Symbolic Regression",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/kstaats/karoo_gp",
+    project_urls={
+        'Documentation': 'https://github.com/kstaats/karoo_gp/blob/master/Karoo_GP_User_Guide.pdf',
+        'Source': 'https://github.com/kstaats/karoo_gp',
+        'Tracker': 'https://github.com/kstaats/karoo_gp/issues',
+    },
+    license="MIT",
+    packages=["karoo_gp"],
+    scripts=["karoo-gp.py"],
+    package_data={
+        'karoo_gp': ['files/*'],
+    },
+    install_requires=[
+        'numpy',
+        'sklearn',
+        'sympy',
+        'tensorflow<2',
+    ],
+    classifiers=[
```

Comment:
from https://pypi.org/

Topic
 	Scientific/Engineering
		Artificial Intelligence

and then my own additions if we can:
	genetic programming
	gp
	genetic algorithm
	machine learning
	evolutionary computation

Intended Audience
	Developers
	Education
	Science/Research